packages/odyssey-react/src/components/Tooltip/Tooltip.module.scss

```
@@ -11,23 +11,25 @@
  */
 
 .root {
-  @include border-radius(var(--TipBorderRadius));
+  @include border-radius(var(--BorderRadius));
 
   display: block;
   visibility: hidden;
   position: absolute;
   z-index: 1;
-  padding-block: var(--TipPaddingBlock);
-  padding-inline: var(--TipPaddingInline);
-  transition: opacity var(--TransitionDuration) var(--TransitionTiming)
-    var(--TransitionDelay);
-  border: var(--TipBorderWidth) solid var(--TipBackgroundColor);
+  padding-block: var(--PaddingBlock);
+  padding-inline: var(--PaddingInline);
+  transition-property: opacity;
+  transition-duration: var(--TransitionDuration);
+  transition-timing-function: var(--TransitionTimingFunction);
+  transition-delay: var(--TransitionDelay);
+  border: var(--BorderWidth) solid var(--BackgroundColor);
```

can we break this apart into multiple styles rather than using the shorthand? Helps with readability with all the custom properties being used here. Also we'll want to theme border style for SIWNext
```suggestion
  border-width: var(--BorderWidth);
  border-style: var(--BorderStyle);
  border-color: var(--BackgroundColor);
```